File: src/main/java/com/gjw/common/innovation/controller/IndexController.java
package com.gjw.common.innovation.controller;

import lombok.extern.slf4j.Slf4j;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.PostMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import javax.annotation.Resource;

@Slf4j
@RestController
@RequestMapping("info")
public class IndexController {

    @GetMapping
    public String getFun() {
        return "get请求返回结果数据";
    }

    @PostMapping
    public String postFun() {
        return "get请求返回结果数据";
    }
}

File: module-two/src/main/java/com/go/demo/controller/IndexController.kt
package com.go.demo.controller

import cn.hutool.core.util.IdUtil
import com.go.common.extension.log
import com.go.common.response.R
import com.go.demo.service.IndexService
import jakarta.annotation.Resource
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/api")
class IndexController {

    @Resource
    private lateinit var indexService: IndexService

    @GetMapping("bulkhead")
    fun extractTextFromImage(): R<String> {
        var id = IdUtil.nanoId()
        var msg = indexService.processOne(id)
        log.info("响应数据: {}", msg)
        return R<String>().apply {
            this.data = msg
        }
    }
}